packages/ethers/src/index.ts

```diff
@@ -41,6 +41,10 @@ export class RPChProvider extends JsonRpcProvider {
     log.verbose("Using SEND", method);
     log.verbose("is sdk ready?", this.sdk.isReady);
 
+    if (!this.sdk.isReady) {
+      await this.sdk.start();
```

Comment:
I tried unit testing on this and they were not as useful as I initially expected. But looking at the function, since it is a promise, there is the possibility of it starting twice.
I think the safest thing to do would be to add a boolean variable representing whether the SDK is in the process of starting and then we would avoid it starting twice.

Reply:
I agree